src/liapunovspectrum.jl

```diff
@@ -1,33 +1,44 @@
 # This file is part of the TaylorIntegration.jl package; MIT licensed
 
 """
-    stabilitymatrix!{T<:Number, S<:Number}(eqsdiff, t0::T, x::Array{T,1},
-        jac::Array{T,2})
-    stabilitymatrix!{T<:Number, S<:Number}(eqsdiff, t0::T, x::Array{Taylor1{T},1},
-        jac::Array{Taylor1{T},2})
+    stabilitymatrix!(eqsdiff!, t0, x, δx, dδx, jac)
```

Comment:
I agree that `dδx` should be used instead of `δdx`, since `dδx` represents the time-derivative of the variations `δx`